tests/acceptance/git-revision-test.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from 'ember-twiddle/tests/helpers/start-app';
+
+module('Acceptance | gist-revision', {
+  beforeEach: function() {
+    this.application = startApp();
+  },
+
+  afterEach: function() {
+    Ember.run(this.application, 'destroy');
+  }
+});
+
+test('Able to load a previous revision of a gist', function(assert) {
+
+  const files = [
+    {
+      filename: "application.template.hbs",
+      content: "Hello, World!"
+    }
+  ];
+
+  runGist({
+    type: "revision",
```

Comment:
To me it looks weird to have a function called `runGist` which you can then pass an optional flag to run something else. Why not create `runGistRevision`?